cmd/zoekt-webserver/main.go

```diff
@@ -138,6 +136,20 @@ func main() {
 
 	flag.Parse()
 
+	logDirFlag := flag.Lookup("log_dir")
```

Comment:
nit: add a comment for why we are doing this. (IE glog is a transitive dep with a conflicting flag)